using-mobile-extensions/adobe-experience-platform-assurance/README.md

```diff
@@ -74,39 +74,119 @@ import AEPAssurance // <-- import the AEPAssurance library
 {% tab title="React Native" %}
 #### React Native
 
-{% hint style="danger" %}
-Updated support for React Native is coming. This section will be updated shortly.
-{% endhint %}
+1. Install Assurance.
+
+   ```bash
+   npm install @adobe/react-native-aepassurance
+   ```
+
+   1.1 Link
+
+   * **React Native 0.60+**
+
+[CLI autolink feature](https://github.com/react-native-community/cli/blob/master/docs/autolinking.md) links the module while building the app.
+
+* **React Native &lt;= 0.59**
+
+```bash
+   react-native link @adobe/react-native-aepassurance
+```
+
+_Note_ For `iOS` using `cocoapods`, run:
+
+```bash
+   cd ios/ && pod install
+```
+
+1. Import the extension.
+
+   ```jsx
+    import {AEPAssurance} from '@adobe/react-native-aepassurance';
+   ```
+
+2. Get the extension version.
+
+   ```jsx
+    AEPAssurance.extensionVersion().then(version => console.log("AdobeExperienceSDK: AEPAssurance version: " + version));
+   ```
+
 {% endtab %}
 
 {% tab title="Flutter" %}
-#### Dart
+#### Flutter
+
+1. Install AEPAssurance.
+
+   Flutter install instructions for AEPAssurance can be found [here](https://pub.dev/packages/flutter_assurance/install). 
+
+2. Import the extension.
+
+   ```dart
+   import 'package:flutter_assurance/flutter_assurance.dart';
+   ```
+
+3. Get the extension version.
+
+   ```dart
+   String version = await FlutterAEPAssurance.extensionVersion;
+   ```
 
-Flutter install instructions for AEPAssurance can be found [here](https://pub.dev/packages/flutter_assurance/install).
 {% endtab %}
 
 {% tab title="Cordova" %}
+
 #### Cordova
 
-{% hint style="danger" %}
-Updated support for Cordova is coming. This section will be updated shortly.
-{% endhint %}
+1. After creating your Cordova app and adding the Android and iOS platforms, the AEPAssurance extension for Cordova can be added with this command:
+
+   ```text
+   cordova plugin add https://github.com/adobe/cordova-aepassurance.git
+   ```
+
+2. Get the extension version.
+
+   ```jsx
+   AEPAssurance.extensionVersion(function(version) {  
+      console.log("AEPAssurance version: " + version);
+   }, function(error) {  
+      console.log(error);  
+   });
+   ```
+
 {% endtab %}
 
 {% tab title="Unity" %}
 #### C\#
 
-{% hint style="danger" %}
-Updated support for Unity is coming. This section will be updated shortly.
-{% endhint %}
+1. After importing the [AEPAssurance.unitypackage](https://github.com/adobe/unity-aepassurance/tree/master/bin), the Assurance extension for Unity can be added with following code in the MainScript
```

Comment:
Add a "the" for "added with the following code".
And, add a : after "MainScript".